gap/CategoryOfQuivers.gi

```diff
@@ -98,6 +98,49 @@ InstallMethodWithCache( CategoryOfQuiversEnrichedOver,
     
     F_hat := FiniteCocompletion( F, category_of_skeletal_finsets : FinalizeCategory := true );
     
+    ModelingCategory( F_hat )!.compiler_hints.category_attribute_resolving_functions := rec(
+        DefiningPairOfUnderlyingQuiver := { } -> rec(
```

Comment:
Yes: `ENHANCED_SYNTAX_TREE( x -> [ 2, [ 0, 1 ], [ 0, 1 ] ] ).bindings.BINDING_RETURN_VALUE`. I have added a comment to the code. In the future this might also be extended by a convenience which automatically resolves literal data types (integers, literal lists, etc.), see my comment here: https://github.com/homalg-project/CAP_project/pull/1111#issue-1422260084. However, I first want to see where and how this system is used.